### ISSUE TITLE
Binding to child elements of the component

### ISSUE THREAD

**Issue by acidjazz** [BKWLD/vue-in-viewport-mixin#7]
I wonder if it would be possible to use this on child elements of the component, similar to the viewport directive you've made where I can just add the directive anywhere.

Maybe for example w/in my component I add a `:in-viewport-child='identifier'` and then my `watch` of `inViewport.now` is passed the "identifier" as a second argument along with `visible`.

I'd definitely find this pretty helpful.

**Comment by weotch** [comment]
Yeah, I like that. It's like are changing the target that is watched. 

I don't see me starting this any time soon, but I'll keep it in mind.

**Comment by acidjazz** [comment]
@weotch Thanks, glad you like the idea. I'll fork this and give it a whirl myself when I get some free time
